README.md

```diff
@@ -0,0 +1,77 @@
+# Book Library Project
+
+[ note for Reviewer:
+Ideally I would have completed more features on this project and written a more complete README. I hope to work on it more before including it in my Portfolio ]
+
+## About
+I created this project as part of the [Manchester Codes](https://www.manchestercodes.com/) software engineering course. The objective was to learn about creating a CRUD API using Express.js and Sequelize.
+
+The API has access to Reader, Book, Genre, and Author tables. Users (Readers) are able to create an entry for themselves and associate books with their account. Genre and Author information is kept in seperate but associated tables, allowing users to search the database by genre or author.
+There is also password authentification, and after a user creates a password it is hidden in the API, and other users are not able to access it. 
+
+## Setup
+The project uses JavaScript, Express.js, Node, and Sequelize.
+Testing is done using Chai, Supertest, and Mocha. 
+
+Download and initial setup: 
+* Fork the project repository
+* Get the project link, navigate to the local folder where the project should be installed, and run ``git clone`` 
```

Comment:
You only need to use one pair of backticks to create code formatting: 

```
* Get the project link, navigate to the local folder where the project should be installed, and run `git clone` 
```

For future reference If you use 3 you will get a code block like this:
```
// some code here
```